js/app.js

```diff
@@ -0,0 +1,23 @@
+// navigation section
+function openNav() {
+  document.getElementById('overlay').style.width = '100%';
+}
+
+function closeNav() {
+  document.getElementById('overlay').style.width = '0%';
+}
```

Comment:
> When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- You have done great so far :+1: :+1:  Your mobile menu looks like the Figma design. But there are some requirements you need to implement before moving to the next milestone. :+1: 

- [ ] As stated in the project requirement, when a user clicks on a menu item `Portfolio, About, Contact`, the mobile menu should also close while taking the user to that part of the website. I'll suggest you implement this. :wave: :+1: